tests/shared/src/test/scala/munit/ScalaCheckSeedSuite.scala

```diff
@@ -15,6 +15,25 @@ final class ScalaCheckSeedSuite extends ScalaCheckSuite {
     }
   }
 
+  test("generated int are not all the same") {
+    println(ints)
+    assert(ints.size > 1)
+  }
+}
+
+final class ScalaCheckInitialSeedSuite extends ScalaCheckSuite {
+
+  // initial seed should be used for the first out of 100 `minSuccessfulTests` only
+  override val scalaCheckInitialSeed = "9SohH7wEYXCdXK4b9yM2d6TKIN2jBFcBs4QBta-2yTD="
```

Comment:
This test checks that when initial seed is overridden, we still generate different input per test run instead of reusing the same.